scaletest/terraform/prometheus.tf

```diff
@@ -1,10 +1,14 @@
 locals {
-  prometheus_helm_repo            = "https://charts.bitnami.com/bitnami"
-  prometheus_helm_chart           = "kube-prometheus"
-  prometheus_helm_version         = null // just use latest
-  prometheus_release_name         = "prometheus"
-  prometheus_namespace            = "prometheus"
-  prometheus_remote_write_enabled = var.prometheus_remote_write_password != ""
+  prometheus_helm_repo             = "https://charts.bitnami.com/bitnami"
+  prometheus_helm_chart            = "kube-prometheus"
+  prometheus_helm_version          = null // just use latest
+  prometheus_exporter_helm_repo    = "https://prometheus-community.github.io/helm-charts"
+  prometheus_exporter_helm_chart   = "prometheus-postgres-exporter"
+  prometheus_exporter_helm_version = null // use latest
```

Comment:
Just checking; would putting a version here lock it down? If not, perhaps we can exclude it so as to not confuse?

Reply:
Docs say no: https://registry.terraform.io/providers/hashicorp/helm/latest/docs/resources/release#version